### DISCUSSION TITLE
v0.6.0

### DISCUSSION THREAD

**Discussion by michenriksen** [michenriksen/hugo-theme-til#5] [Announcements]
### Features
- introduce spoiler shortcode

<hr /><em>This discussion was created from the release <a href='https://github.com/michenriksen/hugo-theme-til/releases/tag/v0.6.0'>v0.6.0</a>.</em>

**Comment by walkbori** [comment]
I wasn't sure where to ask my question, so I'm leaving it here. First, I want to say that your theme is truly amazing. I'm currently testing it to use for my site. 

My question is, how can I change the font? The font you've set is great, but since I use Korean, I’d like to apply a font that's more suitable. I tried modifying the main.css file in the asset/css folder, but the changes didn’t seem to take effect.

Oh, I'm non-developer. Just start to learn Python for hobby and know basic html, css.
